### DISCUSSION TITLE
Azure Email / Communication Implementation

### DISCUSSION THREAD

**Discussion by Arun-Ghosh** [jertel/elastalert2#1575] [Ideas]
Hi All,

Is it possible to implement Azure communication (Azure native mailing, call, whatsapp etc service) in elertalert Alerting. As AWS SES and SNS is there, so it would be good to implement Azure native alerting too :)

**Comment by jertel** [comment]
Hello,

Yes it is possible. You can submit a PR following the [contribution guidelines](../blob/master/CONTRIBUTING.md). Using an existing alerter class as a template will help speed up the process.
